kindlecomicconverter/KCC_gui.py

```diff
@@ -642,6 +642,9 @@ def changeDevice(self):
             self.modeChange(2)
         else:
             self.modeChange(1)
+        if profile['ForceColor']:
+            GUI.colorBox.setChecked(profile.ForceColor)
+            GUI.colorBox.setEnabled(profile.ForceColor)
```

Comment:
What does this code do? The checkbox is already enabled.

Reply:
I came across this while testing my changes by executing `kcc.py`. However, It is indeed not needed anymore, otherwise the checkbox will be completely disabled.